jsontests/src/lib.rs

```diff
@@ -69,121 +70,139 @@ impl Event {
-				self.events.push(Event::SLoad {
-					address: address.clone(),
-					index: index.clone(),
-					value: value.clone(),
-				}),
-        RuntimeEvent::SStore { address, index, value } =>
-				self.events.push(Event::SStore {
-					address: address.clone(),
-					index: index.clone(),
-					value: value.clone(),
-				}),
-      _ => (),
-    };
-  }
+	fn event(&mut self, event: evm_runtime::tracing::Event<'_>) {
+		use evm_runtime::tracing::Event as RuntimeEvent;
+		match event {
+			RuntimeEvent::Step {
+				context,
+				opcode,
+				position,
+				stack,
+				memory,
+			} => self.events.push(Event::Step {
+				sender: context.caller.clone(),
+				contract: context.address.clone(),
+				position: position.clone().unwrap(),
+				opcode: opcode.0,
+				stack: stack.data().clone(),
+				memory: memory.data().clone(),
+			}),
+			RuntimeEvent::SLoad {
+				address,
+				index,
+				value,
+			} => self.events.push(Event::SLoad {
+				address: address.clone(),
+				index: index.clone(),
+				value: value.clone(),
+			}),
+			RuntimeEvent::SStore {
+				address,
+				index,
+				value,
+			} => self.events.push(Event::SStore {
+				address: address.clone(),
+				index: index.clone(),
+				value: value.clone(),
+			}),
+			_ => (),
+		};
+	}
 }
 
 pub fn exit_reason_to_u8(exit_reason: &ExitReason) -> u8 {
-  match exit_reason {
-    evm_runtime::ExitReason::Succeed(s) => s.clone() as u8,
-    evm_runtime::ExitReason::Revert(_) => 0x10,
-    evm_runtime::ExitReason::Fatal(_) => 0x20,
-    evm_runtime::ExitReason::Error(e) => {
-      match e {
-        ExitError::StackUnderflow => 0x30,
-        ExitError::StackOverflow => 0x31,
-        ExitError::InvalidJump => 0x32,
-        ExitError::InvalidRange => 0x33,
-        ExitError::DesignatedInvalid => 0x34,
-        ExitError::CallTooDeep => 0x35,
-        ExitError::OutOfOffset => 0x38,
-        ExitError::OutOfGas => 0x39,
-        ExitError::Other(_) => 0x3d,
-        ExitError::InvalidCode(_) => 0x3f,
-        _ => 0x30,
-      }
-    },
-  }
+	match exit_reason {
+		evm_runtime::ExitReason::Succeed(s) => s.clone() as u8,
+		evm_runtime::ExitReason::Revert(_) => 0x10,
+		evm_runtime::ExitReason::Fatal(_) => 0x20,
+		evm_runtime::ExitReason::Error(e) => match e {
+			ExitError::StackUnderflow => 0x30,
+			ExitError::StackOverflow => 0x31,
+			ExitError::InvalidJump => 0x32,
+			ExitError::InvalidRange => 0x33,
+			ExitError::DesignatedInvalid => 0x34,
+			ExitError::CallTooDeep => 0x35,
+			ExitError::OutOfOffset => 0x38,
+			ExitError::OutOfGas => 0x39,
+			ExitError::Other(_) => 0x3d,
+			ExitError::InvalidCode(_) => 0x3f,
+			_ => 0x30,
+		},
+	}
 }
 
 pub fn run_move_test() -> anyhow::Result<Vec<Event>> {
 	// aptos move test --bytecode-version 6 -i 10000000 -f steps
-	let command_output = Command::new("aptos")
-		.current_dir("/Users/bulent/Desktop/Blockchain/EVM/devm")
+	let command_output = Command::new("/data_disk/duo/aptos-core/target/release/aptos")
```

Comment:
Can we create an .env file and an example.env to have sample parameters?

I think I used `dot_env` package for this in the proxy code.